frontend/components/ArticleBody.tsx

```diff
@@ -68,6 +69,18 @@ const wrapper = css`
         }
     }
 `;
+const pillarColours = pillarMap(
```

Comment:
Both this and below feel like pretty generic behaviour. Happy to keep them here if it's the only use, but would be good to move them into the pillar package at some point.

Reply:
I like the idea of extracting these functions, however I don't like making `css` a dependency of the pillars module. Maybe we can move these functions into a new module called `lib/pillar-style-helpers` or something like that? We could move `pillarMap` there too.

Reply:
Yeah, I put them in `pillars.ts` to start with, I think this forms part of the more general "where does css live" question too.